README.md

```diff
@@ -13,6 +13,38 @@ Our goal is to document these sets of solutions to challenges, make it easy for
 These implementations will likely be documents, videos, or code, each of which is generic enough to be used across companies,
```

Comment:
I added a comment earlier, but noticed your request for me to review it.

Given our recent work, I'd like to conduct a review in a working session.

Reply:
Sounds good.